Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -42,24 +42,30 @@ public interface IConnectionManager : IDisposable
 
     public class ConnectionManager : IConnectionManager
     {
-        // The maximum number of entries in an 'inv' protocol message
-        public const int MAX_INV_SZ = 50000;
+        // The maximum number of entries in an 'inv' protocol message.
+        public const int MaxInventorySize = 50000;
+
+        private readonly ILogger logger;
+        private readonly INodeLifetime nodeLifetime;
 
         private readonly NodesCollection connectedNodes = new NodesCollection();
+        public IReadOnlyNodesCollection ConnectedNodes { get { return this.connectedNodes; } }
```

Comment:
please use` =>` for these.

Reply:
let's leave this as is now, but discuss this in #688

Reply:
ok